src/txHandler/Bulker.TxHandler.ts

```diff
@@ -1,15 +1,194 @@
-import { Operation } from "src/simulation/simulation.types";
+import { BigNumber, constants } from "ethers";
+import { deepCopy, getAddress } from "ethers/lib/utils";
+import { Subject } from "rxjs";
+import { TxOperation } from "src/simulation/simulation.types";
 
+import { WadRayMath } from "@morpho-labs/ethers-utils/lib/maths";
+import { maxBN } from "@morpho-labs/ethers-utils/lib/utils";
+
+import { MorphoAaveV3DataHolder } from "../MorphoAaveV3DataHolder";
+import addresses from "../contracts/addresses";
+import CONTRACT_ADDRESSES from "../contracts/addresses";
 import { TransactionOptions } from "../types";
 
+import { Bulker } from "./Bulker.TxHandler.interface";
 import { NotifierManager } from "./NotifierManager";
-import { IBatchTxHandler } from "./TxHandler.interface";
-
-export default class BulkerTxHandler extends NotifierManager implements IBatchTxHandler {
-  public handleBatchTransaction(
-    operations: Operation[],
-    options?: TransactionOptions | undefined
-  ): Promise<any> {
-    return Promise.reject("TODO: to implement");
+import { IBatchTxHandler, ISimpleTxHandler } from "./TxHandler.interface";
+
+import TransactionType = Bulker.TransactionType;
+
+export default class BulkerTxHandler
+  extends NotifierManager
+  implements IBatchTxHandler
+{
+  _operations: TxOperation[] = [];
+  _bulkerTransactions: Bulker.Transactions[] = [];
+  _value = constants.Zero;
+
+  public readonly operations$ = new Subject<TxOperation[]>();
+
+  protected set operations(ops: TxOperation[]) {
+    this._operations = ops;
+    this.operations$.next(ops);
+  }
```

Comment:
Using a `BehviorSubject` is more efficient (I forgot about it when i developed the adapter)

A behaviorSubjetc is basically a subject with a memory (subject.getValue() returns the last emitted value)

Reply:
you mean no need to have 2 variables?

Reply:
exactly but it can be a global refactoring on the whole adapter i guess